File: tutscodes/python_basic/check_prime_number.py
# Function for checking prime nber
def is_prime_number(n): 
	if n > 1: 
		for val in range(2, n):
			if n % val == 0:
				print("Not Prime", n)
				break
		else:
			print("Prime", n)
	else:
		print("Not Prime", n)

# Check Prime Number
is_prime_number(15)

File: tutscodes/programs/php/check-prime-number.php
<?php
// Number
$number = 138;
$isPrimeNumber = true;

// Using for loop
for ($n = 2; $n < $number; $n++) {
    if($number%$n == 0) {
        $isPrimeNumber = false;
        break;
    }
}

if ($isPrimeNumber) {
    echo "$number is prime number.";
} else {
    echo "$number is prime not number.";
}    